doc/reference/reference_sql/sql_user_guide.rst

```diff
@@ -192,8 +192,8 @@ DOUBLE literals: |br|
 Examples: .0, 1.0, 1E5, 1.1E5. |br|
 A literal has :ref:`data type = DOUBLE <sql_data_type_double>` if it contains a period, or contains "E".
 DOUBLE literals are also known as floating-point literals or approximate-numeric literals.
-To represent "Inf" (infinity), write a real number outside the double-precision number range, for example 1E309.
-To represent "nan" (not a number), write an expression that does not result in a real number,
+To represent "Inf" (infinity), write a real numeric outside the double-precision numeric range, for example 1E309.
+To represent "nan" (not a number), write an expression that does not result in a real numeric,
```

Comment:
@pgulutzan By "real numeric", do you mean a number belonging to [this mathematical set](https://en.wikipedia.org/wiki/Real_number)? That is, to represent "nan", you have to enter square root of minus one, for example?

Reply:
True enough, although since we're talking about a restriction of the set, I prefer to think of the Wikipedia article about https://en.wikipedia.org/wiki/NaN. As described earlier, we avoid the English word "number" now except for the 'number' type, but perhaps in Russian translation there would be no such problem. Is this preventing you from approving the pull request? ~~~~

Reply:
No, but this is preventing me from finishing the translation.

> As described earlier, we avoid the English word "number" now except for the 'number' type
Could you please give me a link to this discussion? I did not know of it.

Reply:
What made the matter more necessary now was that the NUMBER data type no longer had the typical characteristics that one associates with numbers. So in the initial comment I said
"
We will say "numeric" rather than "number" in some places. It's clumsy, but otherwise people will associate "number" with "NUMBER".
The alternative is to say, repeatedly, "number (but not NUMBER)".
"
Since no reviewer objected, I think it's considered okay.